mne/preprocessing/eyetracking/utils.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
+
+from ...utils import _validate_type
+from .calibration import Calibration
+
+
+def _check_calibration(
+    calibration, want_keys=("screen_size", "screen_resolution", "screen_distance")
+):
+    missing_keys = []
+    for key in want_keys:
+        if calibration.get(key, None) is None:
+            missing_keys.append(key)
+
+    if missing_keys:
+        raise KeyError(
+            "Calibration object must have the following keys with valid values:"
+            f" {', '.join(missing_keys)}"
+        )
+    else:
+        return True
+
+
+def get_screen_visual_angle(calibration):
+    """Calculate the radian of visual angle that the participant screen subtends.
```

Comment:
singular ("the radian") on purpose here?

Reply:
An absent minded mistake, thanks for the catch!